### ISSUE TITLE
Tinylog.org only has Javadoc for tinylog-api

### ISSUE THREAD

**Issue by Sailsman63** [tinylog-org/tinylog#359]
**Describe the bug**
The preloaded javadoc on tinylog.org seems to only cover the `tinylog-api`package. This means that core extension points like `Writer` are not documented, and need to be reviewed in source form.

**Comment by pmwmedia** [comment]
Yes, you are right. However, for tinylog 3, the Javadoc on the website will contain all public classes from `tinylog-core`, `tinylog-api`, and `tinylog-impl`. We could use the same Maven commands for tinylog 2,

https://github.com/tinylog-org/tinylog/blob/v3.0/pom.xml#L209
https://github.com/tinylog-org/website/blob/v3/.github/workflows/build.yaml#L61
